demisto_sdk/commands/validate/validators/PB_validators/PB116_is_stopping_on_error.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+def is_indicator_pb(playbook: Playbook):
```

Comment:
@aaron1535 refactor to use this function
@YuvHayun is there a better place to put this? we have 3 validations that should use it

Reply:
Done